app/views/reviews/show.html.erb

```diff
@@ -111,6 +111,37 @@
 </p>
 
 <%= link_to 'Edit', edit_review_path(@review) %> |
-<%= link_to 'Back', professor_path(@review.professor_id) %>
+<%= link_to 'Back', professor_path(@review.professor_id) %> 
+
+
+<button class="btn btn-primary" id="myBtn">Report Review</button>
+
+<div id="myModal" class="modal">
+    <div class="modal-content">
+        <span class="close">&times;</span>
+        <div class ="header">
+          <h1>Report Review</h1>
+            <p>Please let us know why this review is being reported.</p>
+
+        </div>
+        
+        <div class="wrapper-class">
+          <input type="radio" id="radio1">
+               <label for="radio1">It is spam or abusive.</label><br><br>
+          <input type="radio" id="radio2">
+               <label for="radio2">It uses inappropriate or offensive language.</label><br><br>
+           <input type="radio" id="radio3">
+                <label for="radio3">Other</label><br>
+        </div>
+        <input id="other_input" type="text"  placeholder="Please specify for 'Other'">
+
+        <%=link_to "Submit Report", review_path(:sendemail => 'yes'), id: "submitReview", class: "btn btn-primary w-30"%>
+
+    </div>
+</div>
+
+<%= javascript_include_tag "index"%>
+
```

Comment:
Please do not copy-paste code! As you have re-used the exact same code as in the `index` file, you should create a [Rails partial](https://guides.rubyonrails.org/layouts_and_rendering.html#using-partials) for the report dialog, and render that instead. This allows you to modify the report dialog in one file, and have all of the changes everywhere where that partial is used.

See the below command for rendering the footer for more details.